src/test/scala/uk/gov/nationalarchives/tdr/api/routes/FileRouteSpec.scala

```diff
@@ -2,8 +2,10 @@ package uk.gov.nationalarchives.tdr.api.routes
 
 import akka.http.scaladsl.model.headers.OAuth2BearerToken
 import com.dimafeng.testcontainers.PostgreSQLContainer
+import io.circe.Printer
```

Comment:
Is this import needed?

Reply:
No, I was printing the response json to see what was up with the tests. Same with the one below. I've removed them both.